bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -465,13 +454,10 @@ void ReplicaImp::onMessage<PrePrepareMsg>(PrePrepareMsg *msg) {
     Assert(!msg->isNull());  // we should never send (and never accept) null PrePrepare message
 
     if (viewsManager->addPotentiallyMissingPP(msg, lastStableSeqNum)) {
-      LOG_INFO(GL,
-               "Node " << config_.replicaId << " adds PrePrepareMsg for seqNumber " << msgSeqNum << " to viewsManager");
+      LOG_INFO(GL, "PrePrepare added to views manager. " << KVLOG(lastStableSeqNum));
```

Comment:
I think it would be useful to also have the msg sequence number here.

Reply:
It's included in the [Scoped MDC](https://github.com/vmware/concord-bft/pull/638/files/5d4f021d26d0bc3cdd079f39f3b8e7e26a0b06a1#diff-48324f7e49aa046d298836b4ccc2a318L455) so including it would be redundant.